src/general/errors.go

```diff
@@ -0,0 +1,7 @@
+package general
```

Comment:
Hi @egawata since this file is a new addition. We'll need you to put in the license at the top of the file.

Reply:
You can find the license at the beginning of any of the `.go` files

Reply:
@Gituser143 
I'm sorry for forgetting that 🙇‍♂️   I've added it at https://github.com/pesos/grofer/pull/53/commits/d48915bd6d30abf0a7aa2e7c2e73a50ddf01bd5c .